pkg/storage/gcp/saver.go

```diff
@@ -35,7 +37,23 @@ func New(ctx context.Context, cred option.ClientOption) (*Storage, error) {
 		return nil, err
 	}
 	bkt := client.Bucket(bucketname)
-	return &Storage{bucket: bkt}, nil
+
+	u, err := url.Parse(fmt.Sprintf("https://storage.googleapis.com/%s", bucketname))
```

Comment:
can you pass in the URL string here? we can't hard-code to GCS or any specific system

Reply:
The BaseURL method will check for an override env variable and return that if set, this hard coded one is the default for google cloud storage urls.

Reply:
yep this one is fallback uri to default blob store public URI. in case CDN is not configured this needs to be as it is. i was thinking if it should be moved to config or here. and came to understanding with my inner self that it is ok here, as it is bound to a specific storage provider and should not change